src/components/layout/dashboard/contextHeader/ContextHeader.tsx

```diff
@@ -27,8 +27,7 @@ export const ContextHeaderAppBar: React.FC<{ children: ReactNode }> = ({
   children,
 }) => (
   <AppBar
-    component='nav'
-    aria-label='breadcrumbs-nav'
+    component='div'
```

Comment:
Removing `component='nav'` here since it is applied automatically by the MUI Breadcrumbs component.